src/com/carolinarollergirls/scoreboard/core/impl/ClockImpl.java

```diff
@@ -62,6 +63,15 @@ else if (flag != Flag.INTERNAL) {
         if (prop == Value.INVERTED_TIME) {
             return getMaximumTime() - getTime();
         }
+        if (prop == Value.MAXIMUM_TIME && getId() == ID_INTERMISSION) {
+            long duration = DEFAULT_MAXIMUM_TIME;
```

Comment:
This means I can no longer override the maximum time for an intermissions from the operator screen, which is often required when the schedule needs moving forward.